frontend/packages/metal3-plugin/src/utils/index.ts

```diff
@@ -0,0 +1,7 @@
+import { FirehoseResult } from '@console/internal/components/utils';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+export const getLoadedData = (
```

Comment:
This should go to `@console/shared` as well, in a follow up patch is fine.

Reply:
ok, let's move these later. This function is used in many places in kubevirt.

Reply:
#4039